CONTRIBUTING.md

```diff
@@ -4,6 +4,8 @@ Thanks for your interest in improving Storybook! We are a community-driven proje
 
 Please review this document to help to streamline the process and save everyone's precious time.
 
+This guide assumes you're using `yarn` as package manager. You can use `npm` as well, but it would be less deterministic, as we only commit `yarn.lock` to the repo.
```

Comment:
I don't think that we should push this hard. If you use `npm` after this change your experience will be the same as it is now for everybody =) But if you use `yarn`, it'll be better